deployment/network-operator/README.md

```diff
@@ -165,7 +169,7 @@ devicePlugin:
   deploy: true
   reources:
     - name: rdma_shared_device_a
-      devices: [enp1, enp2]
+      ifNames: [enp1, enp2]
```

Comment:
can you add an example that with vendor and deviceID selectors that maps mellanox cx5 under a resource

Reply:
added

Reply:
I meant add an example -> new example that demonstrates it.

you can have this example demonstrate a network operator deployment of device plugin only that creates a single resource which maps to mellanox cx5